submissions/andrewklmn/frogger-game/app.js

```diff
@@ -30,6 +30,7 @@ Actor.prototype.render = function() {
 // Enemies our player must avoid
```

Comment:
This file doesn't belong to this task.
Please fix

Reply:
You are right! This commit is not relevant for this task ... 
But I found this error just after merging the previous task ... 
Could I leave it this way?